Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonCheckedListBox.cs

```diff
@@ -958,6 +958,7 @@ private void LbnSelChange()
         private bool _alwaysActive;
         private bool _forcedLayout;
         private bool _trackingMouseEnter;
+        private float _cornerRoundingRadius, _itemCornerRoundingRadius;
```

Comment:
These rounding things should not be needed on Controls, as they already have them, AND are not part of the themes thing anyway!